source/windows/windows_error_message.c

```diff
@@ -0,0 +1,21 @@
+#include "windows_error_message.h"
+#include <Windows.h>
+#include <aws/io/logging.h>
+
+void aws_win_log_message(
+    enum aws_log_level log_level,
+    aws_log_subject_t subject,
+    char *function_name,
+    DWORD last_error) {
+    WCHAR buffer[512]; // Buffer for text.
+    DWORD dw_chars;
+    dw_chars = FormatMessage(
+        FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, NULL, last_error, 0, (LPSTR)buffer, 512, NULL);
+    AWS_LOGF(
```

Comment:
Ideally I think we'd want to convert the message back to utf-8.  I'm not at all confident that our logging will properly handle wide string arguments.